_episodes/01-intro.md

```diff
@@ -72,6 +72,11 @@ In the examples for this lesson, we'll show the prompt as `$ `.
 Most importantly:
 when typing commands, either from these lessons or from other sources,
 *do not type the prompt*, only the commands that follow it.
+Also note that after you type a command, you have to press the *Enter* key yo execute it.
+
+The prompt is followed by a **text cursor**, a character that indicates the position where your typing will appear. 
+The cursor is usually a flashing solid rectangle, but it can also be an underscore or a pipe. 
```

Comment:
Perhaps here a "flashing or solid block".

I wonder if we can actually put in the actual text characters instead?

Reply:
Done. I was looking for an ascii code or something for the block cursor but could not find any.